index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <p>Hello World!, We are software developers!</p>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+
+    <div id="main">
+      <!-- <div id="title">
+        <h1>Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+      </div> -->
+
+      <div id="img-div">
+        <figure>
+          <img
+          id="image"
+          src="./images/tribute-page-main-image.jpg"
+          alt="Tribute Main Image"
+        />
+        <figcaption id="img-caption">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexi co on
+          how to increase wheat yields- part of his life-long war on hunger.
+        </figcaption>
+        </figure>
+      </div>
+      
+
+      <h3>Here's a time line of Dr. Borlaug's life:</h3>
+
+      <div id="tribute-info">
```

Comment:
I would suggest you put this inside some semantic tags like <section> or use <article>.
This will make your codebase structure more understandable.